client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -261,9 +263,32 @@ const siteMigration: Flow = {
 						);
 					}
 
+					return navigate( STEPS.SITE_MIGRATION_HOW_TO_MIGRATE.slug, {
+						siteId,
+						siteSlug,
+					} );
+				}
+
+				case STEPS.SITE_MIGRATION_HOW_TO_MIGRATE.slug: {
 					// Take the user to the upgrade plan step.
 					if ( providedDependencies?.destination === 'upgrade' ) {
-						return navigate( STEPS.SITE_MIGRATION_UPGRADE_PLAN.slug, {
+						return navigate(
+							addQueryArgs(
+								{
+									siteId,
+									siteSlug,
+									from: fromQueryParam,
+									destination: providedDependencies?.destination,
+									how: providedDependencies?.how as string,
+								},
+								STEPS.SITE_MIGRATION_UPGRADE_PLAN.slug
+							)
+						);
+					}
+
+					// Do it for me option.
+					if ( providedDependencies?.how === HOW_TO_MIGRATE_OPTIONS.DO_IT_FOR_ME ) {
+						return navigate( STEPS.SITE_MIGRATION_ASSISTED_MIGRATION.slug, {
```

Comment:
Was this step added in a previous PR? It would be good to reference it somewhere, as the description implies that it's included in this PR based on the screenshots.

Reply:
Yes, the step was introduced here: https://github.com/Automattic/wp-calypso/pull/89896

I will fix the description.